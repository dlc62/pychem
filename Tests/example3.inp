[example3]
Method = "HF"
Basis_Sets = ["6-311G"]
Multiplicity = 1
Charge = 0
Coords = [['H', 1.0,  0.  ,  0.,   0.],
          ['H', 1.0,  1.0 ,  0.,   0.]]
Reference = "CUHF"
Max_SCF_Iterations = 50

