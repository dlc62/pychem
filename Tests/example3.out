-------------------------------------------------------
 Hartree-Fock iterations 
-------------------------------------------------------

 Cycle:  1
 Total energy:  -1.0615920859
 Change in energy:  0.647182618353

 Cycle:  2
 Total energy:  -1.08951022817
 Change in energy:  -0.0279181422696

 Cycle:  3
 Total energy:  -1.0952681431
 Change in energy:  -0.00575791493448

 Cycle:  4
 Total energy:  -1.09651097175
 Change in energy:  -0.00124282865146

 Cycle:  5
 Total energy:  -1.0967851321
 Change in energy:  -0.000274160345686

 Cycle:  6
 Total energy:  -1.09684653327
 Change in energy:  -6.1401173898e-05

 Cycle:  7
 Total energy:  -1.09686041487
 Change in energy:  -1.38815971296e-05

 Cycle:  8
 Total energy:  -1.09686357001
 Change in energy:  -3.15514568316e-06

 Cycle:  9
 Total energy:  -1.09686428922
 Change in energy:  -7.19202845501e-07

-------------------------------------------------------
 End of Hartree-Fock iterations 
-------------------------------------------------------

<S^2> = 0.00

Final HF energy:  -1.09686428922


 Alpha Orbital Energies 
[-0.52791985  0.12351675  0.32767456  0.55229746  2.53716264  2.65148625]

 Alpha MOs 
[[ 0.16103948  0.10410176  0.04534339 -0.09785354 -1.08632601  1.13312462]
 [ 0.30670056  0.07410439  0.82277374 -1.34614599  1.6429203  -1.21628127]
 [ 0.16692802  1.54967401 -0.88857956  1.86878683 -1.04729928  0.46949982]
 [ 0.16103948 -0.10410176  0.04534339  0.09785354  1.08632601  1.13312462]
 [ 0.30670056 -0.07410439  0.82277374  1.34614599 -1.6429203  -1.21628127]
 [ 0.16692802 -1.54967401 -0.88857956 -1.86878683  1.04729928  0.46949982]]

 Alpha Density Matrix 
[[ 0.02593371  0.0493909   0.026882    0.02593371  0.0493909   0.026882  ]
 [ 0.0493909   0.09406524  0.05119692  0.0493909   0.09406524  0.05119692]
 [ 0.026882    0.05119692  0.02786496  0.026882    0.05119692  0.02786496]
 [ 0.02593371  0.0493909   0.026882    0.02593371  0.0493909   0.026882  ]
 [ 0.0493909   0.09406524  0.05119692  0.0493909   0.09406524  0.05119692]
 [ 0.026882    0.05119692  0.02786496  0.026882    0.05119692  0.02786496]]


 Beta Orbital Energies 
[-0.52791985  0.12351675  0.32767456  0.55229746  2.53716264  2.65148625]

 Beta MOs 
[[ 0.16103948  0.10410176  0.04534339 -0.09785354 -1.08632601  1.13312462]
 [ 0.30670056  0.07410439  0.82277374 -1.34614599  1.6429203  -1.21628127]
 [ 0.16692802  1.54967401 -0.88857956  1.86878683 -1.04729928  0.46949982]
 [ 0.16103948 -0.10410176  0.04534339  0.09785354  1.08632601  1.13312462]
 [ 0.30670056 -0.07410439  0.82277374  1.34614599 -1.6429203  -1.21628127]
 [ 0.16692802 -1.54967401 -0.88857956 -1.86878683  1.04729928  0.46949982]]

 Beta Density Matrix 
[[ 0.02593371  0.0493909   0.026882    0.02593371  0.0493909   0.026882  ]
 [ 0.0493909   0.09406524  0.05119692  0.0493909   0.09406524  0.05119692]
 [ 0.026882    0.05119692  0.02786496  0.026882    0.05119692  0.02786496]
 [ 0.02593371  0.0493909   0.026882    0.02593371  0.0493909   0.026882  ]
 [ 0.0493909   0.09406524  0.05119692  0.0493909   0.09406524  0.05119692]
 [ 0.026882    0.05119692  0.02786496  0.026882    0.05119692  0.02786496]]


-------------------------------------------------------
 End of Hartree-Fock calculation 
-------------------------------------------------------

